core-blocks/image/edit.js

```diff
@@ -52,6 +51,13 @@ import ImageSize from './image-size';
  */
 const MIN_SIZE = 20;
 
+const LINK_DESTINATION_OPTIONS = {
+	NONE: 'NONE',
```

Comment:
Unsure about whether to use all caps for these constant properties. Could also make them individual consts instead of putting them in an object.

Reply:
> Could also make them individual consts instead of putting them in an object.

I'd go with this 🙂